src/kiota/KiotaHost.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using Kiota.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Kiota {
+    public static class KiotaHost {
+        public static RootCommand GetRootCommand()
+        {
+            var configuration = LoadDefaultConfiguration();
+            var outputOption = new Option("--output", "The ouput path of the folder the code will be generated in.") { Argument = new Argument<string>(() => "./output") };
+            outputOption.AddAlias("-o");
+            
+            var languageOption = new Option("--language", "The language to generate the code in.") { Argument = new Argument<GenerationLanguage?>(() => GenerationLanguage.CSharp) };
+            languageOption.AddAlias("-l");
+            AddEnumValidator<GenerationLanguage>(languageOption.Argument, "language");
+            var classOption = new Option("--class-name", "The class name to use the for main entry point") { Argument = new Argument<string>(() => "GraphClient") };
+            classOption.AddAlias("-c");
+            AddStringRegexValidator(classOption.Argument, @"^[a-zA-Z_][\w_-]+", "class name");
+
+            var namespaceOption = new Option("--namespace-name", "The namespace name to use the for main entry point") { Argument = new Argument<string>(() => "GraphClient") };
+            namespaceOption.AddAlias("-n");
+            AddStringRegexValidator(namespaceOption.Argument, @"^[\w][\w\._-]+", "namespace name");
+
+            var logLevelOption = new Option("--loglevel", "The log level to use when logging events to the main output.") { Argument = new Argument<LogLevel>(() => LogLevel.Warning)};
+            logLevelOption.AddAlias("--ll");
+            AddEnumValidator<LogLevel>(logLevelOption.Argument, "log level");
+            var descriptionOption = new Option("--openapi", "The path to the OpenAPI description file used to generate the code.") {Argument = new Argument<string>(() => "openapi.yml")};
```

Comment:
`The path to the OpenAPI description file used to generate the code.` becomes
`The path to the OpenAPI description file used to generate the code files. The default value is openapi.yml. The path is relative to your present working directory.`

Reply:
This accepts both absolutes and relative paths, and I'm worried that by saying relative in the description, people will think it only accepts relative values.